starter-code/src/functions-and-arrays.js

```diff
@@ -1,93 +1,488 @@
 // Iteration #1: Find the maximum
 
+function maxOfTwoNumbers(num1, num2) {
+  if (num1 > num2) {
+    return num1;
+  } else {
+    return num2;
+  }
+}
+
 // Iteration #2: Find longest word
-const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+const words = [
+  "mystery",
+  "brother",
+  "aviator",
+  "crocodile",
+  "pearl",
+  "orchard",
+  "crackpot"
+];
+
+function findLongestWord(arrOfWords) {
+  let longestWord = [];
+  let counter = 0;
+
+  if (arrOfWords.length === 0) {
+    return null;
+  } else {
+    for (let i = 0; i < arrOfWords.length; i++) {
+      if (arrOfWords[i].length > counter) {
+        counter = arrOfWords[i].length;
+        longestWord.unshift(arrOfWords[i]);
```

Comment:
I believe that your solution is a little bit too complicated for this exercise actually. You could always be checking directly the size of the word and, instead of doing an array and unshifting, you could just reassign it.